cmake/CompilerFlags.cmake

```diff
@@ -137,17 +137,19 @@ if(UNIX)
   )
 endif()
 
-register_compiler_flags(
-  DESCRIPTION "Place each function in its own section"
-  -ffunction-sections ${UNIX}
-  /Gy ${WIN32}
-)
+if (NOT ENABLE_LTO)
```

Comment:
Convention in CMake is no spaces between `if` and the statement

Reply:
if you want to enforce stylistic choices, add a formatter